rclpy/package.xml

```diff
@@ -33,6 +33,7 @@
   <depend>rmw</depend>
   <depend>rmw_implementation</depend>
   <depend>rosidl_runtime_c</depend>
+  <depend>type_description_interfaces</exec_depend>
```

Comment:
```suggestion
  <depend>type_description_interfaces</depend>
```

Reply:
Oops, sorry. I tossed off a quick change as I was going out the door. I'll try to get to this this afternoon.